lib/wda/utils.js

```diff
@@ -156,6 +157,47 @@ CODE_SIGN_IDENTITY = ${signingId}
   return xcconfigPath;
 }
 
+/**
+ * Creates xctestrun file per device & platform version.
+ * We expects to have WebDriverAgentRunner_iphoneos${platformVersion}-arm64.xctestrun for real device
+ * and WebDriverAgentRunner_iphonesimulator${platformVersion}-x86_64.xctestrun for simulator located @bootstrapPath
+ *
+ * @param {boolean} isRealDevice - Equals to true if the current device is a real device
+ * @param {string} udid - The device UDID.
+ * @param {string} platformVersion - The platform version of OS.
+ * @param {string} bootstrapPath - The device UDID.
+ * @param {string} wdaRemotePort - The remote port WDA is listening on.
+ */
+async function setXCTESTRUNFile (isRealDevice, udid, platformVersion, bootstrapPath, wdaRemotePort) {
```

Comment:
it might be useful to pass the whole opts objects

Reply:
Today we don't store `opts` in xcodebuild. We create a variable from `args` passed to `xcodebuild` class. So if I have to pass `opts` then i need to create one while invoking this method which would look equally bad. From method signature point of view , its good to know what we are passing instead of just passing `opts`. I think if we really have to pass `opts`, then we probably need create class instead of method or should have this method in class where it is called.